tests/test_tensor.py

```diff
@@ -360,7 +360,7 @@ def test_baddbmm_(self):
 
 
 class PermuteTests(unittest.TestCase):
-    def test_3d(self):
+    def dest3d(self):
```

Comment:
didn't really understand. why changing name didn't work here.
but reverting to the original name worked. so I'm going to leave it as it is.